fission-core/library/Fission/DataRoot.hs

```diff
@@ -0,0 +1,6 @@
+module Fission.DataRoot
```

Comment:
I like it!

A couple things:

`where` flush left? Everywhere else we have it next to the closing `)`.

I like the impulse to break things into classes 🙌 🎉

`DataRoot` is a bit generic here, no? Is it the data root _in general_ on some IPLD, or over on DNS? Perhaps `MonadWNFS`?

```haskell
module Fission.WebNative.FileSystem.Class where

-- ...

class Monad m => MonadWNFS m where
  getDataRootForUser :: m (Either ServerError CID)
```

Reply:
> where flush left? Everywhere else we have it next to the closing ).

that was a formatting tool thing. working on it :ok_hand: I'll override it for now

Reply:
hmmmm yeah I wasn't totally happy with `MonadDataRoot`, I like `MonadWNFS` :+1: